lab-anthony/lib/greet.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+function greet(name){
+  console.log('Hello ' + process.argv[2]);
+  return 'Hello ' + name;
+}
+
+exports.greet = greet;
```

Comment:
Great job!  For future consideration, if your module only exports one function it may be simpler to just return that function directly (module.exports = function(...) {...};) so when you require it you don't have to specify .greet to get what you want.